source/_integrations/sensor.rest.markdown

```diff
@@ -335,40 +330,29 @@ This sample fetches a weather report from [OpenWeatherMap](https://openweatherma
 {% raw %}
 
 ```yaml
-sensor:
-  - platform: rest
-    name: OWM_report
-    json_attributes:
-      - main
-      - weather
-    value_template: "{{ value_json['weather'][0]['description'].title() }}"
-    resource: https://api.openweathermap.org/data/2.5/weather?zip=80302,us&APPID=VERYSECRETAPIKEY
-  - platform: template
-    sensors:
-      owm_weather:
-        value_template: "{{ state_attr('sensor.owm_report', 'weather')[0]['description'].title() }}"
-        entity_picture_template: "{{ 'https://openweathermap.org/img/w/' + state_attr('sensor.owm_report', 'weather')[0]['icon'].lower() + '.png' }}"
-        entity_id: sensor.owm_report
-      owm_temp:
-        friendly_name: "Outside temp"
-        value_template: "{{ state_attr('sensor.owm_report', 'main')['temp'] - 273.15 }}"
+rest:
+  - resource: https://api.openweathermap.org/data/2.5/weather?zip=80302,us&APPID=VERYSECRETAPIKEY
+    sensor:
+      - name: "Report"
+        value_template: "{{ value_json['weather'][0]['description'].title() }}"
+        picture: "{{ 'https://openweathermap.org/img/w/' + value_json['weather'][0]['icon'].lower() + '.png' }}"
+
+      - name: "Outside temp"
+        value_template: "{{ value_json['main']['temp'] - 273.15 }}"
         unit_of_measurement: "°C"
-        entity_id: sensor.owm_report
-      owm_pressure:
-        friendly_name: "Outside pressure"
-        value_template: "{{ state_attr('sensor.owm_report', 'main')['pressure'] }}"
+
+      - name: "Outside pressure"
+        value_template: "{{ value_json['main']['pressure'] }}"
         unit_of_measurement: "hP"
-        entity_id: sensor.owm_report
-      owm_humidity:
-        friendly_name: "Outside humidity"
-        value_template: "{{ state_attr('sensor.owm_report', 'main')['humidity'] }}"
+
+      - name: "Outside humidity"
+        value_template: "{{ value_json['main']['humidity'] }}"
         unit_of_measurement: "%"
-        entity_id: sensor.owm_report
 ```
 
 {% endraw %}
 
-This configuration shows how to extract multiple values from a dictionary with `json_attributes` and `template`. This avoids flooding the REST service by only requesting the result once, then creating multiple attributes from that single result using templates. By default, the sensor state would be set to the full JSON — here, that would exceed the 255-character maximum allowed length for the state, so we override that default by using `value_template` to set a static value of `OK`.
+This configuration shows how to extract multiple values from a dictionary. This avoids flooding the REST service by only requesting the result once, then creating multiple sensors from that single result using template sensors.
```

Comment:
```suggestion
This configuration shows how to extract multiple values from a dictionary. This method avoids flooding the REST service because the result is only requested once. From that single request, multiple sensors can be created by using template sensors.
```